packages/bitcore-node/src/modules/ethereum/models/transaction.ts

```diff
@@ -62,10 +62,10 @@ export class EthTransactionModel extends BaseTransaction<IEthTransaction> {
     this.collection.createIndex({ chain: 1, network: 1, from: 1 }, { background: true, sparse: true });
     this.collection.createIndex({ chain: 1, network: 1, from: 1, nonce: 1 }, { background: true, sparse: true });
     this.collection.createIndex(
-      { chain: 1, network: 1, 'abiType.params.0.value': 1, blockTimeNormalized: 1 },
```

Comment:
Is this required? This index is already created, so changing this would create new indexes, and the previous indexes would be dropped.